script.js

```diff
@@ -0,0 +1,47 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const showBooks = document.getElementById('show-books');
+const submitBtn = document.getElementById('addBtn');
+let books = [];
+
+function display() {
+  document.querySelector('#show-books').innerHTML = '';
+  books.forEach((book) => {
+    showBooks.innerHTML += `
+        <h3>${book.title}</h3>
+        <h3>${book.author}</h3>
+        <button id=${book.id} onClick=removeBook(id) type="submit">Remove</button>
+        <hr>
+        `;
+  });
+  title.value = '';
+  author.value = '';
+}
+
+function addNewBook() {
+  if (title.value.length > 0 && author.value.length > 0) {
+    books.push({
+      title: title.value,
+      author: author.value,
+      id: books.length,
+    });
+    display();
+    addLocalStorage();
+  }
+}
+function removeBook(id) {
+  books = books.filter((e) => e.id != id);
+  display();
+  addLocalStorage();
+}
```

Comment:
- [ ] when you have a list of books and the user removes a book and after that, the user adds another book there can be 2 books with the same id, if the user tries to remove either of these 2 books after that both books will disappear. you can try to fix this by updating the id of all books after removing a book.

Reply:
Thank you for your review. We have worked on that and solved it.